eduNEXT/eox-nelp
# feat: add get_user_data pipeline

## Description

This adds a new pipeline step that returns the basic user data

## Testing instructions

 1. Create a user and complete profile multiple fields 
 2. Go to the pytho shell `./manage.py lms shell`
 3. import the function `from eox_nelp.pearson_vue.pipeline import get_user_data`
 4. Call the function with the id of the user that was created on the step 1. `get_user_data(id)`
 5. Check that the output return the right data
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/e1a731f9-1789-4fb8-ae1f-dda34a577a9f)
